app/models/workflowgroup.rb

```diff
@@ -11,7 +11,8 @@
 
 
 class Workflowgroup < ApplicationRecord
+  acts_as_tenant(:tenant)
```

Comment:
Shouldn't every table in the DB have this feature?

Reply:
@bzwei My understanding is yes. Workflow and Group need to link to each other with same tenant, right?

Reply:
Each id is unique, even without tenant. The purpose of tenant is separation so that one tenant cannot query entries from the other. Workflowgroup is a just a lookup table. You still need the tenant id to retrieve the actual workflow or group entries. Technically we should not need to protect this lookup table unless the `act_as_tenant` gem requires so.

Reply:
I tested. If no tenant is set on a record, all tenants can share it. I'll remove this line.